Makefile.in

```diff
@@ -288,6 +291,10 @@ endif
 
 adler32_test$(EXE): adler32_test.o $(OBJG) $(STATICLIB)
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ adler32_test.o $(OBJG) $(TEST_LIBS) $(LDSHAREDLIBC)
+
+crc32_test$(EXE): crc32_test.o $(STATICLIB)
+	$(CC) $(LDFLAGS) -o $@ crc32_test.o $(TEST_LIBS) $(LDSHAREDLIBC)
+
```

Comment:
This removes executing strip on `adler32_test$(EXE)`!

Reply:
Fixed.